nubis/bin/ldap2consul.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+
+import os
+import sys
+import ConfigParser
+import ldap
+import consul
+
+_config = {}
+
+def load_config(config_file):
+    global _config
+    if not os.path.exists(config_file):
+        print "ERROR: Config file %s not found." % config_file
+        sys.exit(-1)
+    config = ConfigParser.RawConfigParser()
+    config.read(config_file)
+    if 'ldap' not in config.sections():
+        print "ERROR: [ldap] section not found in %s." % config_file
+        sys.exit(-1)
+    _config = config
+
+def getLDAPUsers(ldap_conn):
+    searchbase = _config.get('ldap', 'searchbase')
+    searchfilter = _config.get('ldap', 'searchfilter')
+    try:
+        result_set = []
+        sid = ldap_conn.search(searchbase, ldap.SCOPE_SUBTREE, searchfilter)
+        result_type, result_data = ldap_conn.result(sid, 0)
+        users = result_data[0][1]["member"]
+    except ldap.LDAPError, e:
+        print e
```

Comment:
Yes. The only useful information at this stage, in event of error, is.. the error. What did the LDAP server dislike about our query? What communications problem occurred? etc.